x-pack/plugins/maps/public/routes/list_page/maps_list_view.tsx

```diff
@@ -46,6 +46,7 @@ function navigateToNewMap() {
 const toTableListViewSavedObject = (mapItem: MapItem): MapUserContent => {
   return {
     ...mapItem,
+    namespaces: mapItem.namespaces ?? [],
```

Comment:
I think `namespaces` should always be present on the saved object, so we could also add an assertion operator rather than coalescing to an empty array. WDYT?